chart/kubeapps/templates/kubeappsapis/deployment.yaml

```diff
@@ -74,6 +74,14 @@ spec:
           env:
             - name: PORT
               value: {{ .Values.kubeappsapis.containerPort | quote }}
+            {{- if .Values.clusters }}
+            - name: CLUSTERS_CONFIG_PATH
+              value: /config/clusters.conf
+            {{- end }}
+            {{- if .Values.pinnipedProxy.enabled }}
+            - name: PINNIPED_PROXY_URL
+              value: http://{{ template "kubeapps.pinniped-proxy.fullname" . }}.{{ .Release.Namespace }}:{{ .Values.pinnipedProxy.containerPort }}
+            {{- end }}
```

Comment:
Yep, I did because of my dev setup. I tend to use use the full setup (pinniped + 2 clusters), in fact, sometimes I also use the AKS cluster for testing specific things. 
Perhaps it was simpler to just get the token, but, for me, the crux was on how to pass the authentication from the main program back to the plugins.
Now I have everything more clear, I can split it into two PRs if you want.